alpine/aarch64/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+# set env var
+NODE_VERSION=$1
+ARCH=alpine-aarch64
+TAR_FILE=node-v$NODE_VERSION-linux-$ARCH.tar.gz
+BUCKET_NAME=$BUCKET_NAME
+BINARYNAME=node-v$NODE_VERSION-linux-$ARCH
+
+commit=($(echo "$(grep " v$NODE_VERSION" /commit-table)" | tr " " "\n"))
+if [ -z $commit ]; then
+	echo "commit for v$NODE_VERSION not found!"
+	exit 1
+fi
+
+# compile node
+cd node \
+	&& git checkout ${commit[0]} \
+	&& ./configure --prefix=/ --shared-zlib --shared-openssl \
+   	&& make -j$(nproc) -C out mksnapshot \
+   	&& paxctl -c -m out/Release/mksnapshot \
```

Comment:
@petrosagg, https://git.alpinelinux.org/cgit/aports/tree/main/nodejs/APKBUILD?id=c6acdb11e843e65acf68d71473134149ea4604d3.
I follow the way they build for official package.